src/content/docs/ru/concepts/why-astro.mdx

```diff
@@ -1,135 +1,90 @@
-Вот пять основных принципов проектирования, которые объясняют, почему мы создали Astro,
-проблемы, для решения которых он существует, и почему Astro может быть лучшим выбором для вашего проекта или команды.
+## Особенности
 
-#### Astro ...
+*Astro - это универсальный веб-фреймворк.** Он включает в себя всё необходимое для создания веб-сайта. Также доступны сотни различных [интеграций](https://astro.build/integrations/) и [API-хуков](/ru/reference/integrations-reference/) для настройки проекта под ваши нужды и потребности. 
 
-1. [Ориентирован на контент](#ориентирован-на-контент): Astro был спроектирован для сайтов, с насыщенным контентом.
-2. [Server-first](#server-first): Веб-сайты работают быстрее, когда они рендерят HTML на сервере.
-3. [Быстрый по-умолчанию](#быстрый-по-умолчанию): Создать медленный веб-сайт с Astro должно быть практически невозможно.
-4. [Легкий в использовании](#легкий-в-использовании): Вам не нужно быть экспертом в Astro.
-5. [Многофункциональный, но гибкий](#многофункциональный-но-гибкий): Более 100+ интеграций Astro на выбор.
+Некоторые особенности:
 
-## Ориентирован на контент
+- **[Острова](/ru/concepts/islands/):** Компонентная веб-архитектура, оптимизированная для веб-сайтов, ориентированных на контент.
+- **[Без UI-ограничений](/ru/guides/framework-components/):** Поддерживает React, Preact, Svelte, Vue, Solid, Lit, HTMX, веб-компоненты и многое другое.
+- **[Серверно-ориентированный](/ru/basics/rendering-modes/):** Выносит затратную рендеринговую нагрузку с устройств ваших посетителей.
+- **[Нет JavaScript по умолчанию](/ru/basics/astro-components/):** Меньше JavaScript на стороне клиента, замедляющего работу сайта.
+- **[Коллекции контента](/ru/guides/content-collections/):** Организует, проверяет и обеспечивает типизацию TypeScript для вашего Markdown-контента.
+- **[Настраиваемый](/ru/guides/integrations-guide/):** Tailwind, MDX и сотни интеграций на выбор.
 
-**Astro был спроектирован для сайтов, с насыщенным контентом...** 
-Это включает маркетинговые, новостные сайты, сайты-документации, блоги, портфолио и ecommerce сайты.
+## Принципы дизайна
 
-Напротив же, большинство веб фреймворков предназначены для создания веб-приложений.
-Эти фреймворки лучше всего подходят для создания более сложных приложений в браузере: панелей администратора,
-почтовых сервисов, социальных сетей, списков задач и даже нативных приложений, таких как [Figma](https://figma.com/) 
-и [Ping](https://ping.gg/).
+Вот пять основных принципов дизайна, которые помогут объяснить, почему мы создали Astro, какие проблемы он решает и почему Astro может быть лучшим выбором для вашего проекта или команды.
 
-Это одно из самых важных различий, которые необходимо понять в Astro.
-Уникальный фокус на контент позволяет Astro находить компромиссы и предоставлять непревзойденные функции 
-производительности, которые не имели бы смысла внедрять во веб-фреймворках более ориентированных на веб-приложения.
+Astro...
```

Comment:
There is a typography symbol `…` on current `en` page. By the typography rules of Russian lang the same symbol is required too
```suggestion
Astro…
```

Reply:
Thanks for the review! If you check closer, the English version, or any other, uses 'Astro is. . .' not 'Astro...'. On the website it looks all the same.